<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsTableDialog</class>
 <widget class="QWidget" name="ResultsTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Extra Columns</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="whatsThis">
         <string>The expressions are JavaScript expressions. Additionally, the values from the table columns can be fetched with the following &lt;i&gt;special variables&lt;/i&gt;:

&lt;table border=&quot;0&quot; cellpadding=&quot;3&quot;&gt;
  &lt;tr&gt;&lt;td&gt;&lt;b&gt;LINES[x]&lt;/b&gt;&lt;/td&gt;&lt;td&gt;Lines from table &lt;i&gt;x&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
  &lt;tr&gt;&lt;td&gt;&lt;b&gt;GATES[x]&lt;/b&gt;&lt;/td&gt;&lt;td&gt;Gates from table &lt;i&gt;x&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
  &lt;tr&gt;&lt;td&gt;&lt;b&gt;QC[x]&lt;/b&gt;&lt;/td&gt;&lt;td&gt;Quantum costs from table &lt;i&gt;x&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
  &lt;tr&gt;&lt;td&gt;&lt;b&gt;TC[x]&lt;/b&gt;&lt;/td&gt;&lt;td&gt;Transistor costs from table &lt;i&gt;x&lt;/i&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;/table&gt;

The index variable always starts from 0. When processing the table the respective value from the current row is inserted.
</string>
        </property>
        <column>
         <property name="text">
          <string>Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Expression</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>282</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
